index.html

```diff
@@ -1275,17 +1317,31 @@ <h2 id="objects">Object Functions</h2>
       <p id="keys">
         <b class="header">keys</b><code>_.keys(object)</code>
         <br />
-        Retrieve all the names of the <b>object</b>'s properties.
+        Retrieve all the names of the <b>object</b>'s own enumerable properties.
       </p>
       <pre>
 _.keys({one: 1, two: 2, three: 3});
 =&gt; ["one", "two", "three"]
 </pre>
 
+      <p id="keysIn">
+        <b class="header">keysIn</b><code>_.keysIn(object)</code>
+        <br />
+        Retrieve all the inheritted and own enumerable properties of an <b>object</b>.
```

Comment:
sp: "inheritted" -> "inherited"

"inherited and own enumerable properties" -> "enumerable inherited and own properties"